lib/src/DateRangePicker/date-range-manager.ts

```diff
@@ -41,8 +41,5 @@ export function calculateRangePreview(options: CalculateRangeChangeOptions): Dat
   }
 
   const [previewStart, previewEnd] = newRange;
-  // prettier-ignore
-  return options.currentlySelectingRangeEnd === 'end'
-    ? [end, previewEnd]
-    : [previewStart, start];
+  return options.currentlySelectingRangeEnd === 'end' ? [end, previewEnd] : [previewStart, start];
```

Comment:
Don't like this formatting, it looks broken

Reply:
I think that you make the point for why not using prettier ignore is valuable. It frees us from having discussions around what's the best formatting. Where do we start using it? When do we stop?